next-packages/stylis/types/test.ts

```diff
@@ -74,11 +74,7 @@ stylis.set({
   },
 });
 // $ExpectError
-stylis.set({
-  prefix() {
-    return 'hi';
-  },
-});
+stylis.set({ prefix: () => 'hi' });
```

Comment:
cant both be supported? cc @Ailrun

Reply:
The problem was that TypeScript next reports the error on a different line, one further down. It's still the same error as before.

I can add these lines (👇) as well if you want to? I just changed to a arrow function to make it a bit more condense.

```typescript
// $ExpectError
stylis.set({ prefix () { return 'hi'; } });
```

Reply:
Both supported, but how about just preserving previous test as you mentioned?
I mean, how about just changing it into oneliner?

Reply:
tbh, I think it's better as an arrow function since i think the reason this broke was that I formatted it with prettier and having it as an arrow function means it will stay on one line.

Reply:
Oh, it's changed because of `prettier`... Then alright. Let's leave this as an arrow function.